src/services/transports/types.ts

```diff
@@ -0,0 +1,9 @@
+import { type TransportConfig } from 'viem';
+
+export type LoadBalanceConfig = {
+  key?: TransportConfig['key'];
+  name?: TransportConfig['name'];
```

Comment:
What does key and name do?

Reply:
The Transport requires this by default. It could be hardcoded

Reply:
Let's hardcode it on our side. It doesn't make much sense for users to know about this